client/src/components/images/index.tsx

```diff
@@ -0,0 +1,19 @@
+interface ImageProps {
+  src: string,
+  alt:string
+}
+
+function Image({ src, alt }:ImageProps) {
+  return (
+    <img
+      src={src}
+      alt={alt}
+      style={{
```

Comment:
If we want to use image component every where maybe we should move style from here, OR if we want to use this component in brand component move the file to brand folder